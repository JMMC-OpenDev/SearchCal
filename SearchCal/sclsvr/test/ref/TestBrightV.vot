version https://git-lfs.github.com/spec/v1
oid sha256:67578348e5bfdd4ba3f37f2966c5c6ac9d481deef258fb7629ff34bd3d931441
size 3739715
