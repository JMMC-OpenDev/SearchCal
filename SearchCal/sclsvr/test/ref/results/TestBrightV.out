version https://git-lfs.github.com/spec/v1
oid sha256:c2a7d733108f0a713035937be970f07d2d4c49cbd96832041077fecbd452ead1
size 1826669
